addons/dango-rain/style.css

```diff
@@ -0,0 +1,86 @@
+.sa-dangos-left {
+  height: 100%;
+  position: fixed;
+  top: 0;
+  /* left -50 so if screen is small, dangos can hide offscreen. */
+  left: -50px;
+}
+.sa-dangos-right {
+  height: 100%;
+  position: fixed;
+  top: 0;
+  right: 0;
+}
+.sa-dango-notice {
+  bottom: 20px;
+  position: fixed;
+  z-index: 999;
+  background: #000;
+  left: 20px;
+  border-radius: 10px;
+  padding: 5px;
+}
+
+.sa-dango {
+  color: #fff;
+  font-size: 3em;
+  font-family: Arial, sans-serif;
+  text-shadow: 0 0 5px #000;
```

Comment:
This is unnecessary.

Reply:
maybe without any themes...

Reply:
No, this is completely useless. There is no longer any text in the `::before`s. That styling isn't doing anything.